tests/integration/conftest.py

```diff
@@ -32,6 +34,25 @@ def logging_setup():
         level=logging.DEBUG)
 
 
+@pytest.fixture(scope='session', autouse=True)
+def init_nm_test_interfaces():
+    """
+    Initialize the test interfaces in order to properly operate with existing
+    NetworkManager limitation regarding IPv6 stack.
+    NM is learning the initial IPv6 stack state for an interface when the
+    interfaces is first managed. Therefore, this function makes sure the
+    learned state is IPv6 disabled so NM will return the interfaces to this
+    state when marked with IPv6 method=ignore.
+
+    FIXME: This special handling should be dropped when either NM fixes its
+    behaviour or an explicit disable_ipv6=1 is executed by nmstate.
+    """
+    for ifname in ('eth1', 'eth2'):
+        cmd.exec_cmd(('nmcli', 'device', 'set', ifname, 'managed', 'no'))
```

Comment:
This code should be part of libnmstate and not part of the test.

Reply:
I tried embedding it in libnmstate but it is not possible.
This code makes sure the test env is ready for use by nmstate with the ipv6 limitation in mind.